src/components/bulk-email-tool/api.js

```diff
@@ -2,8 +2,15 @@ import { getConfig } from '@edx/frontend-platform';
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 
 const InstructorApiBaseUrl = `${getConfig().LMS_BASE_URL}/api/instructor/v1`;
+const courseHomeBaseUrl = `${getConfig().LMS_BASE_URL}/api/course_home/course_metadata`;
```

Comment:
Random question. I don't know much (anything) about this particular endpoint. Do you know if there is a difference between the `api/course_home/course_metadata/` endpoint and the `api/course_home/v1/course_metadata` endpoint? Do they both provide the same info? Is one marked for deprecation or anything?

Reply:
AFAIK, they both provide the same data/use the same view code. I just chose the versioned endpoint because I assume that is the most up-to-date and maintained web api. Neither seem depreciated.

Reply:
It doesn't look like we are using the versioned API above for `course_metadata`? I think that API would be the `api/course_home/v1/course_metadata`?

Reply:
oh you're right! Typo -- good catch